DOC fix ungrammatical sentence in doc of SelectFromModel (#19240)
scikit-learn/scikit-learn: doc/modules/feature_selection.rst

Feature selection using SelectFromModel
=======================================

:class:`SelectFromModel` is a meta-transformer that can be used alongside any
estimator that assigns importance to each feature through a specific attribute (such as

> ``coef_``, ``feature_importances_``) or that is callable after fitting.

Reviewer: ```suggestion
``coef_``, ``feature_importances_``) or via an `importance_getter` callable after fitting.
```
